css/pages/authentication.css

```diff
@@ -24,6 +24,17 @@
   font-weight: var(--fw-bold);
 }
 
+.auth-form .input-password-wrapper {
+  position: relative;
+}
+
+.input-password-wrapper i {
+  position: absolute;
```

Comment:
better to have this in utilities

Reply:
Thanks for your suggestion! Have made the changes accordingly